content/en/continuous_integration/tests/_index.md

```diff
@@ -37,22 +37,22 @@ cascade:
+|                                                                                                                                                                                                                  |   .NET  |   Java  |      Javascript      |     Python      |   Ruby  |  Swift  |       JUnit Xml      |
+|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-------:|:-------:|:--------------------:|:---------------:|:-------:|:-------:|:--------------------:|
+| {{< ci-details title="Accurate time/durations results" >}}Microseconds resolution in test start time and duration.{{< /ci-details >}}                                                                            | {{< X >}} | {{< X >}} |        {{< X >}}       |    {{< X >}}    | {{< X >}} | {{< X >}} |                      |
+| {{< ci-details title="Distributed traces on integration tests" >}}Tests that make calls to external services instrumented with Datadog show the full distributed trace in their test details.{{< /ci-details >}} | {{< X >}} | {{< X >}} |        {{< X >}}       |                 | {{< X >}} | {{< X >}} |                      |
+| {{< ci-details title="Agent-based reporting" >}}Ability to report test information through the Datadog Agent.{{< /ci-details >}}                                                                                 | {{< X >}} | {{< X >}} |        {{< X >}}       |    {{< X >}}    | {{< X >}} | {{< X >}} |                      |
+| {{< ci-details title="Agentless reporting" >}}Ability to report test information without the Datadog Agent.{{< /ci-details >}}                                                                                   | {{< X >}} | {{< X >}} |        {{< X >}}       |    {{< X >}}    | {{< X >}} | {{< X >}} |        {{< X >}}       |
+| {{< ci-details title="Test suite level visibility" >}}Visibility over the whole testing process, including session, module, suites and tests. Read more in Test suite level visibility.{{< /ci-details >}}       | {{< X >}} | {{< X >}} |        {{< X >}}       |    {{< X >}}    |         | {{< X >}} |        {{< X >}}       |
+| {{< ci-details title="Manual API" >}}Ability to programmatically create CI Visibility events for test frameworks that are not supported by Datadog's automatic instrumentation.{{< /ci-details >}}               | {{< X >}} | {{< X >}} |        {{< X >}}       |                 |         | {{< X >}} |                      |
+| {{< ci-details title="Codeowner by file" >}}Automatic detection of the owner of a source file based on the CODEOWNERS file.{{< /ci-details >}}                                                                   | {{< X >}} | {{< X >}} |        {{< X >}}       |    {{< X >}}    |         | {{< X >}} |                      |
```

Comment:
let's leave it as codeowner by test, even though it's not exactly correct, because I believe we mention this in multiple places. We can leave the extra color in the explanation though 
```suggestion
| {{< ci-details title="Codeowner by test" >}}Automatic detection of the owner of a test file based on the CODEOWNERS file.{{< /ci-details >}}                                                                   | {{< X >}} | {{< X >}} |        {{< X >}}       |    {{< X >}}    |         | {{< X >}} |                      |
```